internal/kafka/internal/api/admin/private/api/openapi.yaml

```diff
@@ -355,6 +356,11 @@ components:
         max_data_retention_size:
           description: Maximum data storage available to this Kafka
           type: string
+        suspended:
+          description: boolean value indicating whether kafka should be suspended
```

Comment:
Could we add some information about what does setting a kafka as suspended mean from the user's point of view?

Reply:
@miguelsorianod - I added a bit more info there